src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -217,4 +196,39 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
             this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(methodId)),
         ]);
     }
+
+    private _buttonClickEvent(methodId: string): (event: Event) => Promise<InternalCheckoutSelectors> {
+
+        return (event: Event) => {
+            if (event) {
+                event.preventDefault();
```

Comment:
`event?.preventDefault()` rather than `if(event) ...`
Also, if event might not be present, maybe better to mark it as optional in the function signature?